packages/commonwealth/client/scripts/views/components/component_kit/cw_wallets_list.tsx

```diff
@@ -253,6 +258,7 @@ export const CWWalletsList = (props: WalletsListProps) => {
                         .WalletAccount;
                       if (!app.chain.apiInitialized) {
                         await app.chain.initApi();
+                        redraw();
```

Comment:
You realize `redraw()` is a no-op, right?

Reply:
I am realizing now that because @affordances triggers a setState within this onClick event handler. We should not even need a force redraw() because react should automatically rerender based on the state change.

Reply:
`redraw()` is not a force redraw -- it's a no-op. `this.redraw()` would force it in the interop layer ClassComponent.

Reply:
I don't think we can call this.redraw() however because this isn't a ClassComponent?

Reply:
Sure -- if it's not, then we should use a `setState` method to update data + trigger redraws, as in #2778. But the `redraw()` function _does nothing_.

Reply:
I see. So we will refactor these pages to use React.useState. Then trigger this within the onClick